README.md

```diff
@@ -1,6 +1,6 @@
 # Kinesis Batch Sink Plugin
 
-[![cm-available](https://cdap-users.herokuapp.com/assets/cm-available.svg)](https://docs.cask.co/cdap/current/en/integrations/cask-market.html)
+[![cm-available](https://cdap-users.herokuapp.com/assets/cm-available.svg)](https://docs.cdap.io/cdap/current/en/integrations/cdap-market.html)
```

Comment:
`https://docs.cdap.io/cdap/current/en/integrations/cask-market.html`

Reply:
Are we still calling it cask market? @bdmogal